spec/controllers/projects_controller_spec.rb

```diff
@@ -124,9 +124,29 @@
   describe "#destroy" do
     it "deletes the project" do
       expect {
-        delete :destroy, params: {id: project.id}
+        delete :destroy, params: {id: project.id, project: {title: project.title}}
       }.to change(Project, :count).by(-1)
     end
+
+    it "deletes stripped project's title" do
+      project.update(title: " foo bar ")
+      expect {
+        delete :destroy, params: {id: project.id, project: {title: "foo bar"}}
+      }.to change(Project, :count).by(-1)
+    end
+
+    it "deletes stripped project's params" do
+      project.update(title: "foo bar")
+      expect {
+        delete :destroy, params: {id: project.id, project: {title: "foo bar  "}}
+      }.to change(Project, :count).by(-1)
+    end
+
+    it "does not delete the project" do
+      expect {
+        delete :destroy, params: {id: project.id}
+      }.not_to change(Project, :count)
+    end
```

Comment:
Hey @JuanVqz , can we also add a spec for testing that it does not delete when we pass the title of the project but it is not the right title?

Reply:
I have added the spec since it was not something of a big change.

Reply:
Thanks! I've added that test as feature :+1: 
https://github.com/fastruby/points/pull/310/files#diff-b4cf077b77d7765559bc95291a6e424d712c59271a5d6bcc7eb79b2cd3726e87R59